File: ariarh-middleoffice/src/main/metamodel/com/itpeac/ariarh/middleoffice/domain/Conversation_.java
package com.itpeac.ariarh.middleoffice.domain;

import javax.annotation.Generated;
import javax.persistence.metamodel.ListAttribute;
import javax.persistence.metamodel.SingularAttribute;
import javax.persistence.metamodel.StaticMetamodel;

@Generated(value = "org.hibernate.jpamodelgen.JPAMetaModelEntityProcessor")
@StaticMetamodel(Conversation.class)
public abstract class Conversation_ {

	public static volatile SingularAttribute<Conversation, Candidate> candidate;
	public static volatile ListAttribute<Conversation, Message> messagesList;
	public static volatile SingularAttribute<Conversation, Long> id;

	public static final String CANDIDATE = "candidate";
	public static final String MESSAGES_LIST = "messagesList";
	public static final String ID = "id";

}

File: ariarh-webapp/src/main/web/src/app/demo/model/Conversation.ts
import {Candidate} from "./Candidat";
import {Message} from "./Message";

export class Conversation {
    id?: number | undefined;
    candidate: Candidate | undefined;
    messagesList?: Message[] | undefined

    constructor(json?: Conversation
    ) {
        if (json != null) {
            this.id = json.id;
            this.candidate = json.candidate;
            this.messagesList = json.messagesList;
        }
    }
}
